addons/sky_3d/src/TimeOfDay.gd

```diff
@@ -318,6 +327,12 @@ func _get_date_time_os() -> void:
 	set_day(date_time_os.day)
 	set_month(date_time_os.month)
 	set_year(date_time_os.year)
+	
+	
+func update_datetime_display() -> void:
```

Comment:
I think this should be private, and prefaced with _. The user doesn't need to call this. The expectation is that it's automatic.